tables/guardduty_finding_log/guardduty_finding.go

```diff
@@ -0,0 +1,141 @@
+package guardduty_finding_log
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/service/guardduty/types"
+	"github.com/turbot/tailpipe-plugin-sdk/schema"
+)
+
+type GuardDutyFinding struct {
+	schema.CommonFields
+
+	AccountId     *string                   `json:"account_id"`
+	Arn           *string                   `json:"arn"`
+	Description   *string                   `json:"description"`
+	Id            *string                   `json:"id"`
+	Partition     *string                   `json:"partition"`
+	Region        *string                   `json:"region"`
+	SchemaVersion *string                   `json:"schema_version"`
+	Severity      *float64                  `json:"severity"`
+	Title         *string                   `json:"title"`
+	Type          *string                   `json:"type"`
+	CreatedAt     time.Time                 `json:"created_at"`
+	UpdatedAt     *time.Time                `json:"updated_at"`
+	Service       *GuardDutyFindingService  `json:"service,omitempty"`
+	Resource      *GuardDutyFindingResource `json:"resource,omitempty"`
+}
+
+type GuardDutyFindingService struct {
```

Comment:
```suggestion
type Service struct {
```

Do we need to prefix each struct with `GuardDutyFinding` if we're already in the `guardduty_finding.go` file?